.github/workflows/tests.yml

```diff
@@ -75,13 +80,25 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+        go: ["1.18.x"]
         suite: ["cluster", "standalone"]
         backend: ["dir", "btrfs", "lvm", "zfs"]
+        include:
+          - go: stable
+            suite: cluster
+            backend: dir
+          - go: stable
+            suite: standalone
+            backend: dir
```

Comment:
Can we add tip for dir as well?
https://github.com/actions/setup-go/issues/21#issuecomment-997208686